CalibCalorimetry/EcalLaserCorrection/src/EcalLaserDbService.cc

```diff
@@ -192,6 +199,9 @@ float EcalLaserDbService::getLaserCorrection(DetId const& xid, edm::Timestamp co
     p_f = apdpnpair.p3;
   }
 
+  if (t > timestamp.t3.value() + (deltaT_safety_ * oneSecond))
+    t_laser = ((long long)timestamp.t3.value()) + deltaT_safety_ * oneSecond;
```

Comment:
the `* oneSecond` should be removed, either by converting seconds to `long long` 64bits CMSSW times with a 32 bit shifts as soon as the parameter is retrieved, or by doing a `<<32` shift here